config/manifests/bases/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     containerImage: ':container-image:'
     createdAt: ':created-at:'
     description: Network flows collector and monitoring solution
+    operatorframework.io/cluster-monitoring: "true"
```

Comment:
Does having this label only makes sense for `DOWNSTREAM_DEPLOYMENT`? cc @jotak

Reply:
Yes, correct, everything that is downstream specific in CSV goes there: https://gitlab.cee.redhat.com/netobserv-midstream/network-observability-operator-cpaas/-/blob/network-observability-1.2.0-rhel-9/distgit/containers/network-observability-operator-bundle/patch_csv.py